api/src/main/java/org/openmrs/module/ugandaemrreports/library/Moh105CohortLibrary.java

```diff
@@ -614,16 +650,16 @@ public CohortDefinition individualsWhoReceivedHIVTestResultsInLast12Months() {
      * Individuals Tested for the first time
      * @return CohortDefinition
      */
-    public CohortDefinition individualsTestedFirstTime() {
```

Comment:
@solemabrothers Why delete the old definition which is used in previous reports? They are now broken. Just create a new definition and leave the old one untouched